book/preface.md

```diff
@@ -45,8 +45,7 @@ to the implementation you have built, the book says "your browser".
 [^jrwilcox-idea]: This idea is from [J.R. Wilcox][jrw], inspired in
 turn by [S. Zdancewic][sz]'s course on compilers.
 
-The code in this book [uses Python
-3](blog/why-python.md),\index{Python} and we recommend you follow
```

Comment:
just change this to https://browser.engineering/blog/why-python.html instead.

Reply:
Done, but I used the substack link instead.